### ISSUE TITLE
Changes required to run it on CPU

### ISSUE THREAD

**Issue by gourango01** [cheng6076/Variational-LSTM-Autoencoder#5]
Getting following error while running the code on cpu with setting flag **gpuid** to -1

 Error:   invalid arguments: DoubleTensor LongTensor 
expected arguments: [*DoubleTensor*] DoubleTensor DoubleTensor
 at 	VLSTM-Autoencoder.lua:270: in main chunk
**local _, loss = optim.adam(feval, params, optim_state)**
